Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml

```diff
@@ -128,8 +128,7 @@
     pocket1: DoubleEmergencyOxygenTankFilled
     pocket2: BaseUplinkRadio40TC
     belt: ClothingBeltMilitaryWebbing
-  #inhand: # DeltaV - Prevent commander spawning with the declaration of war
-  #  - NukeOpsDeclarationOfWar
+  inhand: NukeOpsDeclarationOfWar
```

Comment:
```suggestion
  inhand: 
      - NukeOpsDeclarationOfWar
```
Maintainer L rizz failed to skibidi.

I'm on mobile, indent may be wrong, please check the hyphen is aligned with the h

Reply:
it isnt :trollface: